### DISCUSSION TITLE
All issues wrongly appear as subscribed for repos that you have set custom notifications

### DISCUSSION THREAD

**Discussion by darkred** [community/community#7206] [Mobile]
E.g. this repo https://github.com/violentmonkey/violentmonkey of which I have chosen to only be notified for new Releases, 
all issues are wrongly displayed as already subscribed. I attach various screenshots to see what I mean:
<details>
<summary>My custom watch settings for the repo (Watch | Custom | Releases):</summary>

![2021-11-03_161744](https://user-images.githubusercontent.com/723651/140078821-99081dff-1012-48ee-88df-a1afd3d10dce.jpg)
</details>

<details>
<summary>Two issue examples (the former subscribed, the latter not) -screenshots from website-:</summary>

![2021-11-03_161856](https://user-images.githubusercontent.com/723651/140078841-264feda9-97fe-4fa4-81a4-6086f89b5b84.jpg)
![2021-11-03_161943](https://user-images.githubusercontent.com/723651/140078862-4ba946ab-5da8-4d9c-882e-e5239a564283.jpg)
</details>

<details>
<summary>How these issues appear in the app (the latter should not appear as subscribed):</summary>

![Screenshot_2021-11-03-16-20-56-226_com github android~](https://user-images.githubusercontent.com/723651/140078692-912a3e5b-1536-47ae-89d2-5210110532e9.jpg)
![Screenshot_2021-11-03-16-21-08-275_com github android~](https://user-images.githubusercontent.com/723651/140078709-31f5ff24-ef09-43a2-8b13-8c802a6bbec3.jpg)
</details>

This doesn't occur with discussions. 

&nbsp;&nbsp;
Using GitHub Mobile 1.39.0 beta on Android 11 .

**Comment by brianlovin** [comment]
Thanks for the note @darkred. There is a subtle difference between the repository Watch menu, and the individual notification subscriptions on an issue or discussion. It's possible to have the Watch button for `Releases only`, but still get notifications on individual issues if you are a participant, or created the issue.

I might have misunderstood your original message, but let me know if that makes sense 😊

**Comment by darkred** [comment]
Yes, that difference is clear to me.

Let me clarify my OP: of the two issues in it (1364 and 1414), I have manually clicked "Subscribe" only for the former, 1364,
and so its ![2021-11-09_000654](https://user-images.githubusercontent.com/723651/140825863-5598bbc4-7add-4b0e-a845-5dbe5344140e.jpg) became: ![2021-11-09_000704](https://user-images.githubusercontent.com/723651/140825877-b054effb-f3f0-4cae-80e3-a07f69449553.jpg)
But, inside the mobile app, issue 1414 also appears as subscribed, which is wrong:
![](https://user-images.githubusercontent.com/723651/140078709-31f5ff24-ef09-43a2-8b13-8c802a6bbec3.jpg)
The same occurs with ALL issues in the repo: they all appear as subscribed, regardless of whether I have manually clicked "Subscribe" for them or not. 
In other words, all issues have this icon: ![2021-11-09_002628](https://user-images.githubusercontent.com/723651/140828230-20f9aa7c-887d-459d-9ebf-cf36ac5fe98b.jpg) none has this: ![2021-11-09_002618](https://user-images.githubusercontent.com/723651/140828233-79ca9da8-4ddf-4522-a294-964716a284f7.jpg)

&nbsp;
The only setting that I have found in the violentmonkey repo that's modified(not in its default state) 
is that I have selected have the Watch button for `Releases only`.
And I'm saying this, because, for other repos where the Watch button is at its default state `Participating and @mentions`:
![2021-11-09_001154](https://user-images.githubusercontent.com/723651/140826464-afde86b2-3fce-4c75-a265-76549cd2f730.jpg)
any issues that I don't have manually set to watch, have the correct icon, ![](https://user-images.githubusercontent.com/723651/140828233-79ca9da8-4ddf-4522-a294-964716a284f7.jpg), in the mobile app.

**Reply by brianlovin** [reply]
On the issues that show you are currently subscribed, are you the author or participant in the thread? You will be automatically subscribed to issues if you create or reply to them.

**Comment by darkred** [comment]
No, neither of them. For example, in issue 1414 from OP, I'm not the author, nor have I commented.

**Reply by brianlovin** [reply]
Okay, thanks. I'm not sure exactly what's going on here, but if you unsubscribe from 1414, does it trigger the correct response and reflect that you're unsubscribed on both platforms afterwards?

**Comment by darkred** [comment]
I just found exactly what's happening:
**the first time** I tap the icon in the app, the subscribe status on web remains unchanged.
From the 2nd time I tap it and on, the subscribe status on web is changed ok every time, as expected.
And this only occurs on repos I have set custom notifications (e.g. 'Releases' only).

&nbsp;
To further clarify my OP example:

on WEB, initially e.g. issue 1414 is unsubscribed:
![](https://user-images.githubusercontent.com/723651/140825863-5598bbc4-7add-4b0e-a845-5dbe5344140e.jpg)

But on the APP it wrongly appears subscribed (![](https://user-images.githubusercontent.com/723651/140828230-20f9aa7c-887d-459d-9ebf-cf36ac5fe98b.jpg)), so I tap the the previous icon, and it it becomes ![](https://user-images.githubusercontent.com/723651/140828233-79ca9da8-4ddf-4522-a294-964716a284f7.jpg)
and I also get an "Unsubscribed from this thread" popup ([screenshot](https://user-images.githubusercontent.com/723651/140969132-311783e5-ef07-4629-9f6e-f74634c3eb68.jpg)) at that time,

but on WEB the first time I tap the icon, the subscribe status remains the same, i.e. as it initially was (unsubscribed):
![](https://user-images.githubusercontent.com/723651/140825863-5598bbc4-7add-4b0e-a845-5dbe5344140e.jpg)

The above only occurs **the first time** I tap the icon in the app.
The second time (and following times) I tap the icon, the subscribe status on web is reflected ok, i.e. it toggles statuses ok.

**Reply by brianlovin** [reply]
Okay, that sounds like a bug — I'm going to try my best to reproduce this issue and get this in front of our engineering team to debug. Thank you for all the details and screenshots here @darkred 🙏

**Comment by Susko3** [comment]
Apologies for the bump, but this has become a functional issue, not only a cosmetic one. In version 1.104.0 I'm unable to subscribe to issues as described in the workaround [here](https://github.com/community/community/discussions/7206#discussioncomment-1614227):

> The second time (and following times) I tap the icon, the subscribe status on web is reflected ok, i.e. it toggles statuses ok.

Tapping on the unsubscribe option in the three-dot menu will unsubscribe, but it will remain in the unsubscribed state -- won't toggle the status as described above.

This forces me to leave the app and subscribe to the issue in my web browser.
